pkg/scheduler/scheduler.go

```diff
@@ -109,12 +110,20 @@ func WithBindTimeoutSeconds(bindTimeoutSeconds int64) Option {
 	}
 }
 
+// WithPodMaxBackoffDurationSeconds sets podMaxBackoffDurationSeconds for Scheduler, the default value is 10.
```

Comment:
The other similar functions also have the default value in their comments

```go
// WithBindTimeoutSeconds sets bindTimeoutSeconds for Scheduler, the default value is 100
```

Reply:
I agree with @ahg-g, given that default values are in a different location.

Reply:
doesn't make it right, we have a constant defining the default value, we should avoid declaring it in other places, even in comments to avoid confusion.

Reply:
> doesn't make it right, we have a constant defining the default value, we should avoid declaring it in other places, even in comments to avoid confusion.

I don't think that there is right or wrong about this, it's only about the trade-off. 

+ if we put a default value copy here, we have to pay attention to update the default value in multiple files when it changes.
+ if we don't put it here, and use a reference here, the developer can be a little harder to find the default value for it.

The chance of using another default value is small, and we already have a convention for this (document it in the comments), so I'd like to put the default value here.

If you still have concerns about this, I could send a PR and discuss it. Either way, I don't think we should do that in this one. WDYT?

Reply:
> if we don't put it here, but use a reference or something else, the developer can be a little harder to find the default value for it.

Not an issue at all for me compared to possible inconsistency across the code, but I don't want to dwell over this.